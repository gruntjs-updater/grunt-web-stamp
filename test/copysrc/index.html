<link rel="stylesheet" href="bulk.css"/>