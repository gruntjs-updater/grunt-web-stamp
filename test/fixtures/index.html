<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"/>
<title>Test</title>
<link href ="test.css" rel="stylesheet"/>
<link href ="missing{key}.css" rel="stylesheet"/>
<link href="  test.css?mk=0  " rel="stylesheet"/>
<script src="mk.js" type="text/javascript"></script>
<script src=" //mk.js" type="text/javascript"></script>
<script src=" about:blank " type="text/javascript"></script>
<script src="#" type="text/javascript"></script>
<script src="mk.js?t=90" type="text/javascript"></script>
<style>
    .p{
        background: url(img/p.png);
    }
</style>
</head>
<body>
    <img src="img/p.png" alt="no"/>
</body>
</html>
