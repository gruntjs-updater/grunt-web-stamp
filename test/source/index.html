<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"/>
<title>Test</title>
<link href ="static/css/test.css" rel="stylesheet"/>
<link href ="missing{key}.css" rel="stylesheet"/>
<link href="  static/css/test.css?mk=0  " rel="stylesheet"/>
<script src="static/js/mk.js" type="text/javascript"></script>
<script src=" //mk.js" type="text/javascript"></script>
<script src=" about:blank " type="text/javascript"></script>
<script src="#" type="text/javascript"></script>
<script src="static/js/mk.js?t=90" type="text/javascript"></script>
<style>
    .p{
        background: url(static/img/p.png);
    }
</style>
</head>
<body>
    <img src="static/img/p.png" alt="no"/>
</body>
</html>
