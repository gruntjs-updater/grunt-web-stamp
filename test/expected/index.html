<!--normal relative link path-->
<link href ="static/css/test.css?t=709632" rel="stylesheet"/>
<!--normal absolute path-->
<link href ="http://p0.css.cdn.com/static/css/test.css" rel="stylesheet"/>
<!--missing path-->
<link href ="missing{key}.css" rel="stylesheet"/>
<!--path with spaces inside-->
<link href="static/css/test.css?mk=0&t=709632" rel="stylesheet"/>
<!--normal relative script path-->
<script src="static/js/mk.js?t=941952" type="text/javascript"></script>
<!--path with leading doubel /-->
<script src="//mk.js" type="text/javascript"></script>
<!--path is about:blank-->
<script src=" about:blank " type="text/javascript"></script>
<!--path is #-->
<script src="#" type="text/javascript"></script>
<!--path with stamp-->
<script src="static/js/mk.js?t=90" type="text/javascript"></script>
<!--path in url-->
background: url(static/img/p.png);
<!--path in img element-->
<img src="static/img/p.png"/>
